src/Illuminate/Database/Query/Builder.php

```diff
@@ -2729,6 +2730,24 @@ public function insert(array $values)
         );
     }
 
+    /**
+     * Evaluate the values of default attributes passed if closures exist.
+     *
+     * @param  array  $attributes
+     * @return array
+     */
+    private function resolveInsertDefalts(array $attributes)
+    {
+        return collect($attributes)
```

Comment:
There is no need for a Collection here just use `array_map` for this

Reply:
Point taken. Thanks